examples/pigweed-app/nrfconnect/main/main.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <array>
+#include <span>
+#include <string_view>
+
+#include "app_config.h"
+#include "LEDWidget.h"
+
+#include <dk_buttons_and_leds.h>
+#include <logging/log.h>
+#include "pw_sys_io/sys_io.h"
+#include "pw_sys_io_nrfconnect/init.h"
+
+LOG_MODULE_REGISTER(app);
+
+LEDWidget sStatusLED;
```

Comment:
Put in anynomous namespace or add static?

Reply:
Done